apps/api/src/app/bridge/bridge.controller.ts

```diff
@@ -31,9 +31,11 @@ import { GetBridgeStatus } from './usecases/get-bridge-status/get-bridge-status.
 import { GetBridgeStatusCommand } from './usecases/get-bridge-status/get-bridge-status.command';
 import { CreateBridgeRequestDto } from './dtos/create-bridge-request.dto';
 import { CreateBridgeResponseDto } from './dtos/create-bridge-response.dto';
+import { UserAuthentication } from '../shared/framework/swagger/api.key.security';
 
 @Controller('/bridge')
 @UseInterceptors(ClassSerializerInterceptor)
+@UserAuthentication()
```

Comment:
This is equal when used on top of controller
`@UserAuthGuard()` === `@UserAuthentication()`
which is equal to using `@UseGuards(UserAuthGuard)` on each controller method

I used `@UserAuthentication()` because its used already on 80% of controllers, whereas `@UserAuthGuard()` was used on the rest 20%